.eslintrc.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
- [ ] I also noticed you have not made any meaningful commit messages. it is good practice to ensure your commit messages are meaningful.
- [ ]  Your PR is not descriptive - include more details to support evidence of the requirements of the project